EIPS/eip-7957.md

```diff
@@ -0,0 +1,70 @@
+---
+eip: 7957
+title: EVM64 - EOF support
+description: EVM Object Format support for EVM64
+author: Wei Tang (@sorpaas)
+discussions-to: https://ethereum-magicians.org/t/eip-7937-64-bit-mode-evm-opcodes-evm64/23794
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-05-26
+requires: 3540, 3670, 4200, 7937
+---
+
+## Abstract
+
+This EIP defines EOF support for EVM64 with its additional code validation rules and `RJUMPI`, `RJUMPV` opcodes.
+
+## Motivation
+
+EOF defines a stricter stricter code validation rule to improve efficiency. Due to EVM64 using multibyte opcode (the mode opcode `C0`), a small adaptation is needed. This EIP also additionally defines a 64-bit mode `RJUMPI` and `RJUMPV` to be 64-bit.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+We define the following gas cost constant:
+
+* `G_RJUMPIV64`: 3
```

Comment:
its better to form a table for all constants here including the codes for various rjumps, and also put 3 in `` as well as other constant values so that its more visible

Reply:
@sorpaas let me know if you want to make this change?